starboard/evergreen/shared/launcher.py

```diff
@@ -271,6 +275,43 @@ def _StageTargetsAndContentsRaspi(self):
     os.makedirs(os.path.join(target_staging_dir, 'lib'))
     shutil.copy(target_binary_src, target_binary_dst)
 
+  def _StageTargetsAndContentsRdk(self):
+    """Stage targets and their contents for GN builds for RDK platforms."""
+    content_subdir = os.path.join('usr', 'share', 'cobalt')
+
+    # TODO(b/296633713):
```

Comment:
This bug is marked resolved.

I understand the context here a bit better now and it seems this is not a TODO. Can you confirm that there are no plans to fix the hardcoded file name in the RDK loader?

The comment could be a bit more concise, e.g.:

`The rdk loader always loads libcobalt.so. Hard code target_name to match.`